lib/src/utils.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:cryptography/cryptography.dart';
+import 'package:solana/src/base58/base58.dart' as base58;
+import 'package:solana/src/curve25519/curve25519.dart';
+import 'package:solana/src/encoder/encoder.dart';
+
+typedef HashFunc = List<int> Function(List<int> m);
+
+abstract class Utils {
```

Comment:
Dart isn't Java, we don't need an abstract class to group static methods. These methods can be top-level functions.

Reply:
I wish it had namespaces so that we could have more semantic meaning in the places where we call them. Like identifying a package. I just wanted more organized code. I don't like the top level functions to be exported directly like that. But if it is the dart way, let's do it the dart way.

Reply:
You can use named imports for that, e.g.

```dart
import 'smth' as utils;

utils.doSmth();
```

Reply:
Yes, I am aware of it and in fact we are doing it inside the library with base58 encoder/decoder functions. I think it's more idiomatic so I will update it.